src/nuclide/skip.project.json

```diff
@@ -26,8 +26,53 @@
     },
     "skip_server": {
       "kind": "Program",
-      "sources": ["skipServer.sk"],
-      "references": [ { "path": "../project", "name": "analyze"  } ]
+      "sources": [
+        "skipServer.sk",
+        "../native/hhvm",
```

Comment:
You may want to create a skip.project.json file inside of ../native and reference it instead of copy pasting all the paths 2 times

Reply:
Yeah, seems like each subdir of src/ should have a project file.

Reply:
Thanks for the feedback. I will merge that and open an issue.

Reply:
#58